src/PowerSystems.jl

```diff
@@ -212,13 +212,15 @@ export get_dynamic_components
 export solve_powerflow!
 export solve_powerflow
 export validate_connectivity
+export calc_connected_components
```

Comment:
if we are keeping it, can we remane to `calculate` or `get`. It is preferable to have full word in exported functions.

Reply:
I left it as `calc` because it's now redefinition of the function borrowed from PowerModels